site/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta.adoc

```diff
@@ -82,7 +94,12 @@ image::identity-providers/idp-initiated/fusionauth-view-identity-provider.png[Vi
 
 Navigate to [breadcrumb]#Settings -> System -> CORS#.
 
-Determine the hostname and scheme of the Okta Identity Provider HTTP POST URL. If the URL is `https://example.okta.com/app/generic-saml/11111151wmJ3HKZYD5d6/saml2`, then the hostname and scheme are `https://example.okta.com`.
+Determine the hostname and scheme of the Okta Identity Provider Single Sign-On URL. If the URL is `https://trial-6089629.okta.com/app/trial-6089629_test_1/exk1mn2r8kVgtAv6q697/sso/saml` then the hostname and scheme are `https://example.okta.com`. Additionally, you might have to add another entry with a trailing backslash `https://example.okta.com/`.
+
+[NOTE]
+====
+If you run into CORS configuration challenges, the best solution is to review the `System > Event Log`. This will tell you exactly what hostname and schemes require a whitelist.
```

Comment:
```suggestion
If you run into CORS configuration challenges, review the [breadcrumb]#System > Event Log# in the FusionAuth admin UI. This will tell you exactly what hostname and schemes must be added to the CORS allow lists.
```